src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -93,6 +99,9 @@
 
         az vm create -g group-name -n vm-name --admin-username deploy  \\
           --image debian --secrets "$vm_secrets"
+  - name: Import a certificate into KeyVault. (autogenerated)
+    text: "az keyvault certificate import --file cert_file\n\nsecrets=$( --name vaultname --vault-name vaultname"
```

Comment:
Screwed up syntax.

Reply:
Yaml dumper we use just decides to add these on it's own. Fixing this.